Alignment/OfflineValidation/macros/FitPVResiduals.C

```diff
@@ -698,10 +727,10 @@ void FitPVResiduals(TString namesandlabels,
 
     if (gDirectory->GetListOfKeys()->Contains("pTinfo")) {
       gDirectory->GetObject("pTinfo", thePtInfoHistos[i]);
-      thePTBINS[i] = thePtInfoHistos[i]->GetBinContent(1) * 3. / thePtInfoHistos[i]->GetEntries();
```

Comment:
What was the original purpose of multiplying the bin content by 3?

Reply:
3 bins in the input histogram.